app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationAdapter.kt

```diff
@@ -71,6 +71,8 @@ class ConversationAdapter(
         AccountId(conversation.address.toString())
     else null
 
+    private val expandedMessageIds = mutableSetOf<Long>()
```

Comment:
Sorry the change to use messageId is not enough, this set needs to have the right type too:

```suggestion
    private val expandedMessageIds = mutableSetOf<MessageId>()
```